src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -307,6 +388,7 @@ private String buildOAuthRedirectUrl() {
      * This is where the user comes back to at the end of the OpenID redirect ping-pong.
      */
     public HttpResponse doFinishLogin(StaplerRequest request) throws IOException {
+    	LOGGER.fine("Realm do finish " + request);
```

Comment:
I'm asking more within the context of Jenkins plugins.

On Tue, Aug 9, 2016 at 8:24 AM, David Eads <notifications@github.com> wrote:

> In src/main/java/org/openshift/jenkins/plugins/openshiftlogin/
> OpenShiftOAuth2SecurityRealm.java
> <https://github.com/openshift/jenkins-openshift-login-plugin/pull/2#discussion_r74047177>
> :
>
> > @@ -307,6 +388,7 @@ private String buildOAuthRedirectUrl() {
> >       * This is where the user comes back to at the end of the OpenID redirect ping-pong.
> >       */
> >      public HttpResponse doFinishLogin(StaplerRequest request) throws IOException {
> > +    	LOGGER.fine("Realm do finish " + request);
>
> Are logging messages supposed to prefix by the method that is calling
> them? It's inconsistently applied in your debugging.
>
> Best practice is usually:
>
> if logger.isloggable(level) logger.logp(level, class, method, format, args)
>
> That does a cheap check before building args for the logging method. And
> the logging method doesn't have to perform an expensive stack inspection to
> determine the caller class/function.
>
> If all you want is a, "did I call" point, you can use logger.entering.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/jenkins-openshift-login-plugin/pull/2/files/2c7cc23ad8dea53bbdfaf093ea54ea5d7d854700#r74047177>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p2ItQcx675gfxn6dnHN1twfJpPZ8ks5qeHF0gaJpZM4Jfdh->
> .
>